controllers/hosts.go

```diff
@@ -354,9 +355,7 @@ func deleteHostFromNetwork(w http.ResponseWriter, r *http.Request) {
 
 	runUpdates(node, false)
 	go func() { // notify of peer change
-		if err := mq.PublishPeerUpdate(); err != nil {
-			logger.Log(1, "error publishing peer update ", err.Error())
-		}
+		go mq.BroadcastDelPeer(currHost, network)
```

Comment:
why is there a second go routine here when updateNode has the same set of calls with no extra goroutine.  Should be consistent.